bitnami/kubeapps/templates/kubeops/service.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
 spec:
   type: ClusterIP
   ports:
-    - port: {{ .Values.kubeops.service.port }}
+    - port: {{ .Values.kubeops.service.ports.http }}
       targetPort: http
       protocol: TCP
       name: http
```

Comment:
Missing extraPorts

Reply:
I believe it is fine right now with just the frontend. If we start seeing use cases where this might be necessary, let's add them.

Reply:
I agree with that 👍